Sofa/framework/DefaultType/src/sofa/defaulttype/TemplatesAliases.cpp

```diff
@@ -42,17 +43,18 @@ TemplateAliasesMap& getTemplateAliasesMap()
 
 bool TemplateAliases::addAlias(const std::string& name, const std::string& result, const bool doWarnUser)
 {
-	TemplateAliasesMap& templateAliases = getTemplateAliasesMap();
-	if (templateAliases.find(name) != templateAliases.end())
+    TemplateAliasesMap& templateAliases = getTemplateAliasesMap();
+    if (templateAliases.find(name) != templateAliases.end())
```

Comment:
I would be better to emit the warning in the RegisterTemplateAlias constructor using the returned bool to detect registration failure.

Reply:
This part of the PR is just code formatting. Indentation was a mess. So I won't touch it in this PR.